Components/Rating/rating.css

```diff
@@ -0,0 +1,12 @@
+.rating-container input {
+  display: none;
+}
+
+.rating-container label {
+  cursor: pointer;
+  color: gold;
+}
+
+.rating-container input:not(:checked) + label {
+  color: grey;
```

Comment:
use colors from color variable instead of declaring them directly

Reply:
Done